script.module.enum34/lib/enum/LICENSE

```diff
@@ -0,0 +1,32 @@
+Copyright (c) 2013, Ethan Furman.
```

Comment:
use this as your license.txt

Reply:
I don't understand what you want me to do.
This is the license from the original author of the enum package which has been integrated into Python Versions 3.4+